src/components/Reusables/ConfirmationPage/ConfirmationPage.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Page,
+  PageContent,
+  PageHeader,
+  List,
+  BorderStyle,
+  PageColor,
+} from '@altinn/altinn-design-system';
+import type { Key } from 'react';
+
+import type { DelegableApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveApi } from '@/rtk/features/delegableApi/delegableApiSlice';
+import { softRemove as softRemoveOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { useAppDispatch } from '@/rtk/app/hooks';
+import type { DelegableOrg } from '@/rtk/features/delegableOrg/delegableOrgSlice';
+import { ReactComponent as OfficeIcon } from '@/assets/Office1.svg';
+import { ReactComponent as SettingsIcon } from '@/assets/Settings.svg';
+import { CompactDeletableListItem } from '@/components/Reusables/CompactDeletableListItem';
+
+import classes from './ConfirmationPage.module.css';
+
+export interface ConfirmationPageProps {
+  firstListItems: DelegableApi[];
+  secondListItems: DelegableOrg[];
```

Comment:
Since these are of type DelegatableApi and DelegatableOrg, can't they just be called apiList and orgList (I don't actually think you need the Items-part)